From: Michael Olbrich <m.olbrich@pengutronix.de>
Date: Sat, 31 Jan 2026 15:42:44 +0100
Subject: [PATCH] task_queue_base: add missing include

Without this, building with gcc 15 fails with:

../webrtc-audio-processing-1.3/webrtc/api/task_queue/task_queue_base.h:53:32: error: 'uint32_t' has not been declared                                                               ../webrtc-audio-processing-1.3/webrtc/api/task_queue/task_queue_base.h:53:32: note: 'uint32_t' is defined in header '<cstdint>'; this is probably fixable by adding '#include <cstdint>'
---
 webrtc/api/task_queue/task_queue_base.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/webrtc/api/task_queue/task_queue_base.h b/webrtc/api/task_queue/task_queue_base.h
index 90b1efd31ebe..7403ee971e37 100644
--- a/webrtc/api/task_queue/task_queue_base.h
+++ b/webrtc/api/task_queue/task_queue_base.h
@@ -11,6 +11,7 @@
 #define API_TASK_QUEUE_TASK_QUEUE_BASE_H_
 
 #include <memory>
+#include <cstdint>
 
 #include "api/task_queue/queued_task.h"
 #include "rtc_base/system/rtc_export.h"
